### PR TITLE
MA-2214 Fix project after evaluate error

### PR DESCRIPTION
**Pull request by midery**

### CODE REVIEW

File: subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class CodeOwnershipValidationPlugin : Plugin<Project> {
         val strictOwnership = target.getBooleanProperty("avito.ownership.strictOwnership", false)
 
         target.afterEvaluate {
-            if (strictOwnership) {
+            if (strictOwnership && it.state.failure == null) {
```
Comment (dsvoronin) [inline subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: i have strongest deja-vu in my life that i already fixed it right on this line.
WTF ?:!)

Comment (midery) [reply subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: Yep, you're right. I just forgot to include this change to a new plugin that doesn't require `subprojects`: https://github.com/avito-tech/avito-android/pull/1414. DRY in a nutshell :) 

We will remove old configuration in CodeOwnershipPlugin with code duplication in the next infra release.

Comment (sergioboi) [reply subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: @midery made a disrespect and deleted this

Comment (RuslanMingaliev) [reply subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: @dsvoronin not only you, WTF???

Comment (RuslanMingaliev) [reply subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: https://github.com/avito-tech/avito-android/pull/1393/files

Comment (midery) [reply subprojects/gradle/code-ownership/plugin/src/main/kotlin/com/avito/android/CodeOwnershipValidationPlugin.kt:14]: Well, as I said: I forgot to include this change to a new plugin. Old plugin is working as intended. 
There was an issue with outdated local develop branch, I suppose. Now it's fixed, sorry for your trouble!

**Review by RuslanMingaliev — APPROVED** [review]

**Pull request merged**